cmd/relay-gossip/README.md

```diff
@@ -86,14 +86,21 @@ drand-relay-gossip run -grpc-connect=127.0.0.1:3000 \
                        -insecure
 ```
 
+Or, with a hashlist:
+```shell
+ drand-relay-gossip run -grpc-connect=127.0.0.1:3000 \
+                       -insecure \
+                       -hash-list=6093f9e4320c285ac4aab50ba821cd5678ec7c5015d3d9d11ef89e2a99741e83,dbd506d6ef76e5f386f41c651dcb808c5bcbd75471cc4eafa3f4df7ad4e4c493
+```
+
 ### Relay HTTP
 
 The gossip relay can also relay directly from an HTTP API. You can specify multiple endpoints to enable failover.
 
 ```sh
-drand-relay-gossip run -url=http://127.0.0.1:3002 \
-                       -url=http://127.0.0.1:3102 \
-                       -hash=6093f9e4320c285ac4aab50ba821cd5678ec7c5015d3d9d11ef89e2a99741e83
+drand-relay-gossip run -url=https://api.drand.sh \
+                       -url=https://api2.drand.sh \
+                       -hash=dbd506d6ef76e5f386f41c651dcb808c5bcbd75471cc4eafa3f4df7ad4e4c493
```

Comment:
```suggestion
                       -hash-list=dbd506d6ef76e5f386f41c651dcb808c5bcbd75471cc4eafa3f4df7ad4e4c493
```
Although the HTTP relay _will_ accept `-hash`, it's deprecated so we should prob update the docs to reflect that

Reply:
It might be deprecated in that relay. In this one I kept it working as it was.
Let me know if I should refactor to deprecate it for the gossip relay too.

Reply:
Ah okay wait I got a little confused by the fact it said `### Relay HTTP` above and thought this applied to the HTTP relay, but it's the HTTP relay of the gossip relay oops!